src/modules/outputs/builtins/lint-bash/entrypoint.sh

```diff
@@ -0,0 +1,27 @@
+# shellcheck shell=bash
+
+function main {
+  local tmp
+  local paths=__envTargets__
+  local args=(
+    "--exclude=SC1090,SC1091,SC2016,SC2153,SC2154"
+    "--external-sources"
+  )
+  # SC1090: Can't follow non-constant source. Use a directive to specify location.
+  # SC1091: Not following: x: openBinaryFile: does not exist (No such file or directory)
+  # SC2016: Expressions don't expand in single quotes, use double quotes for that.
+  # SC2153: Possible misspelling: x may not be assigned, but y is.
+  # SC2154: x is referenced but not assigned.
+
+  info Linting bash code \
+    && tmp="$(mktemp)" \
+    && for path in "${paths[@]}"; do
+      find "${path}" -wholename '*.sh' -type f | sort --ignore-case > "${tmp}" \
+        && while read -r path; do
+          info Linting "${path}" \
+            && shellcheck "${args[@]}" "${path}"
+        done < "${tmp}"
```

Comment:
`|| return 1` to control the exit code

remember that the exit code of a loop is the exit code of the last command in the bucle